src/cascadia/TerminalSettingsEditor/Appearances.idl

```diff
@@ -56,7 +56,7 @@ namespace Microsoft.Terminal.Settings.Editor
         ProfileViewModel SourceProfile;
         static Windows.UI.Xaml.DependencyProperty AppearanceProperty { get; };
 
-		Boolean ShowIndistinguishableColorsItem { get; };
+        Boolean ShowIndistinguishableColorsItem { get; };
```

Comment:
fwiw we don't need this in Cascadia at all; it never builds with the `WindowsInbox` branding

Reply:
Thanks for the callout, removed